app/assess/routes.py

```diff
@@ -105,7 +105,11 @@ def display_sub_criteria(
         score_list = get_score_and_justification(
             application_id, sub_criteria_id, score_history=True
         )
-        latest_score = score_list.pop(0) if len(score_list) > 0 else None
+        latest_score = (
+            score_list.pop(0)
+            if (score_list is not None and len(score_list) > 0)
+            else None
+        )
```

Comment:
```suggestion
        latest_score = next(iter(score_list or []), None)
```

Reply:
this has introduced another error so will investigate in another ticket, for now leaving as is 👍🏾